evaluate.py

```diff
@@ -731,9 +736,9 @@ def __init__(self, shape_dict, layout="NCHW", preproc=None):
 
         self.inputs = {name : image}
 
-    def Validate(self, m, ref_outputs=[]):
+    def Validate(self, m, ref_outputs=[], data=[]):
         if isinstance(m, tvm.runtime.vm.VirtualMachine) or isinstance(m, tvm.runtime.profiler_vm.VirtualMachineProfiler):
-            tvm_output = m.get_outputs()[0]
+            tvm_output = m.invoke("main", **data)
```

Comment:
Yes, we only need it for VM profiler. We should check for "args.debug" to be True, otherwise we should call get_outputs(). Thank you, i will fix that.